docs/rules/eol-last.md

```diff
@@ -34,3 +34,7 @@ function doSmth() {
 }
 // spaces here
 ```
+
+### Options
```

Comment:
What kind of example would you expect? The option only takes effect while fixing, so it is not apparent in the example.